docs/book/src/reference/metrics.md

```diff
@@ -92,5 +94,28 @@ func init() {
 ```
 
 You may then record metrics to those collectors from any part of your
-reconcile loop, and those metrics will be available for prometheus or
+reconcile loop.
+
+For example:
+```
+func (r *MemcachedReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	...
+	...
+	...
+
+	// Add metrics
```

Comment:
```suggestion
```

Technically, the custom metrics can be incremented/modified anywhere in the operator. So I'd say lets remove this from here and just add a note that these custom metrics can be evaluated anywhere in the operator code.

Another thing which could be added to doc, would be the missing roles/role bindings which was causing rbac error. That is explained briefly in the previous paragraph, an example of the cluster role binding which is to be added can be included.